src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -71,7 +71,7 @@ public function copy($originFile, $targetFile, $override = false)
 
             if (is_executable($originFile)) {
                 // User Executable | Group Executable | Other Executable
-                chmod($targetFile, fileperms($targetFile) | 0x0040 | 0x0008 | 0x0001);
+                chmod($targetFile, fileperms($targetFile) | 0100 | 0010 | 0001);
```

Comment:
But this is wrong: you can't allow execution for user+group+other. You should preserve what the original has.